src/enclave-agent/Cargo.toml

```diff
@@ -29,8 +29,18 @@ ttrpc = { git = "https://github.com/haosanzi/ttrpc-rust", features = ["async"] }
 oci-distribution = { git = "https://github.com/krustlet/oci-distribution.git", rev = "f44124c" }
 
 [features]
-default = ["image-rs/enclave-cc"]
-simple = ["image-rs/snapshot-unionfs", "image-rs/signature-cosign", "image-rs/signature-simple", "image-rs/keywrap-native", "image-rs/encryption"]
+default = ["cc-kbc-rustls-tls"]
+
+# confidential resource injection using sample-kbc
+simple = ["image-rs/snapshot-unionfs", "image-rs/signature-cosign-rustls", "image-rs/signature-simple", "image-rs/keywrap-native", "image-rs/encryption-ring", "image-rs/oci-distribution-rustls"]
+
```

Comment:
I created this dummy `simple` target just to be able to run CI tests without having RATS-TLS installed. We can probably drop this completely and move the Rust CI checks to use `cc-kbc-*`feature instead.

`sample-kbc` is always enabled, btw.

Reply:
> sample-kbc is always enabled, btw.

Yes. Now a separate `simple` feature and a `sample-kbc` target helps to build an enclave-agent binary that does not need to run in true sgx libos, though I am ok either abondon `simple` or not

Reply:
FYI: if we abondon `sample-kbc`, we might not need a SIM test in CI. Instead we only need two, s.t. `cc-kbc` and `eaa-kbc`

Reply:
Good point about `simple` not requiring the LibOS, let's just keep it then. I prefer to have `sample_kbc` available because it allows us to run basic CI checks using Github actions without additional setup steps, like in the operator repo: https://github.com/confidential-containers/operator/blob/fe952c8b65e9ab76fc6e963e83caf1a7310627c5/.github/workflows/e2e.yaml#L54-L63